app/controllers/projects_controller.rb

```diff
@@ -62,7 +62,14 @@ def show
 
     @cloned_project = Project.select(:id, :user_id, :title).where(id: @project.cloned_from).first
     @liked_by_cur_user = Like.find_by_user_id_and_project_id(current_user, @project.id)
-    @data_sets = @project.data_sets.includes(:user).select('id', 'title', 'user_id', 'key', 'created_at', 'contributor_name').search(params[:search])
+    @all_data_sets = @project.data_sets.includes(:user).select('id', 'title', 'user_id', 'key', 'created_at', 'contributor_name').search(params[:search])
+
+    # The comparission validates the per_page display. The minimum and maximum are arbitrarily chosen but reasonable
```

Comment:
While you're at it, might as well fix spelling

Reply:
Good idea